control-plane/p4RuntimeSerializer.cpp

```diff
@@ -951,9 +951,44 @@ class P4RuntimeAnalyzer {
             }
         }
 
+        // @platform_property annotation
+        for (auto *annotation : decl->getAnnotations()->annotations) {
```

Comment:
This seems like a different annotation but is bundled into the `addPkgInfo` call? Maybe add some explanation why this is the case?

Reply:
Oh, good point. Because it's part of PkgInfo. Adding.